packages/contentful--create-contentful-app/src/constants.ts

```diff
@@ -4,3 +4,7 @@ export const EXAMPLES_REPO_URL = 'https://github.com/contentful/apps/tree/master
 export const TEMPLATES = ['javascript', 'typescript', 'vue', 'vite-react', 'nextjs'] as const;
 export const EXAMPLES_PATH = 'contentful/apps/examples/';
 export const CONTENTFUL_APP_MANIFEST = 'contentful-app-manifest.json';
+export const IGNORED_CLONED_FILES = [
+  CONTENTFUL_APP_MANIFEST,
+  `package.json`
```

Comment:
is there a package-lock.json file as well? what's the context here?

Reply:
nope - just `package.json` as it's combined into the root folder's package.json